draft-ietf-ppm-dap.md

```diff
@@ -2084,64 +2084,47 @@ aggregation parameter in advance. For example, for Prio3 the only valid
 aggregation parameter is the empty string. For these reasons, the collection
 job is handled asynchronously.
 
-Upon receipt of a `CollectionReq`, the Leader begins by checking that it
+Upon receipt of a `CollectionJobReq`, the Leader begins by checking that it
 recognizes the task ID in the request path. If not, it MUST abort with error
 `unrecognizedTask`.
 
 The Leader MAY further validate the request according to the requirements in
 {{batch-validation}} and abort with the indicated error, though some conditions
 such as the number of valid reports may not be verifiable while handling the
-CollectionReq message, and the batch will have to be re-validated later on
+`CollectionJobReq` message, and the batch will have to be re-validated later on
 regardless.
 
-If the Leader finds the CollectionReq to be valid, it immediately responds with
-HTTP status 201.
-
-The Leader then begins working with the Helper to aggregate the reports
-satisfying the query (or continues this process, depending on the VDAF) as
-described in {{aggregate-flow}}.
-
 Changing a collection job's parameters is illegal, so further requests to
 `PUT /tasks/{tasks}/collection_jobs/{collection-job-id}` for the same
```

Comment:
```suggestion
`PUT /tasks/{task-id}/collection_jobs/{collection-job-id}` for the same
```

(not caused by this PR -- aligns with other uses of parameterized URL path fragments)